src/nearbyinteraction.cs

```diff
@@ -231,6 +239,13 @@ interface NIDeviceCapability {
 		[Abstract]
 		[Export ("supportsCameraAssistance")]
 		bool SupportsCameraAssistance { get; }
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
```

Comment:
hmm macOS availability seems a bit off..
it _is_ available according to headers: 

`@property (nonatomic, readonly) BOOL supportsExtendedDistanceMeasurement API_AVAILABLE(ios(17.0), watchos(10.0), macos(14.0), tvos(17.0));`

but might be safer to not include support for mac as [online docs](https://developer.apple.com/documentation/nearbyinteraction/nidevicecapability/4302251-supportsextendeddistancemeasurem?language=objc) say otherwise and there was no xtro todo either..

Reply:
it is not being added because the protocol ins not included in NIDeviceCapability.
```csharp
[Watch (9, 0), NoTV, NoMac, iOS (16, 0), MacCatalyst (16, 0)]
```

It is safe to just remove mac, but it is not included. Funny that the cecil tests did not complain.
```suggestion
		[Watch (10, 0), NoTV, NoMac, iOS (17, 0), MacCatalyst (17, 0)]
```